src/dune_rules/expander.ml

```diff
@@ -909,6 +909,24 @@ let expand_and_eval_set t set ~standard =
   Ordered_set_lang.eval set ~standard ~eq:String.equal ~parse:(fun ~loc:_ s -> s)
 ;;
 
+module Unordered (Key : Ordered_set_lang.Key) = struct
```

Comment:
Can this code be moved to where it's being used? E.g. `modules_field_evaluator`? I don't see how it depends on any private api's of expander.